dlk/python/dlk/core/operators.py

```diff
@@ -56,6 +55,8 @@ def __init__(self,
         self._check_consistency()
         self._rank = len(shape)
         self._available_buffer = ''
+        self._visited = False
+        self._prop_details = Dict
```

Comment:
Also, Does it need ?

Reply:
Probably not, old code I forgot to delete. I will delete it. Thank you!